rcon/vote_map.py

```diff
@@ -349,7 +349,7 @@ def handle_vote_command(self, rcon, struct_log: StructuredLogLineType) -> bool:
         message = struct_log.get("sub_content", "").strip()
         config = VoteMapConfig()
         enabled = config.get_vote_enabled()
-        if not message.startswith("!votemap"):
+        if not message.startswith(("!votemap", "!vm", "!VOTEMAP", "!VM")):
```

Comment:
Use `message.lower().startswith(..)` to avoid having to excplicitly state all cases

Reply:
Nice one. Changing it now !